asteroid/masknn/convolutional.py

```diff
@@ -175,17 +175,17 @@ def forward(self, mixture_w):
             :class:`torch.Tensor`:
                 estimated mask of shape [batch, n_src, n_filters, n_frames]
         """
-        batch, n_filters, n_frames = mixture_w.size()
+        batch, _, n_frames = mixture_w.size()
```

Comment:
What this related to the jitability?

Reply:
Oops, really sorry for that. As you have certainly guessed, it has nothing to do with jitability :s. Is there any way to rewrite the commit history without making it worse?

Reply:
It's better to leave unrelated code unchanged. This is fine for this one, you can leave it.